dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/RelationshipRestRepository.java

```diff
@@ -228,4 +233,35 @@ protected void delete(Context context, Integer id) throws AuthorizeException {
             log.error("Error deleting Relationship specified by ID:" + id, e);
         }
     }
+
+    @SearchRestMethod(name = "byLabel")
+    public Page<RelationshipRest> findByLabel(@Parameter(value = "label", required = true) String label,
```

Comment:
Javadoc is missing here (new public method).

Reply:
Good catch
I've asked Raf to include Javadoc here as well